### PR TITLE
gridea: Add version 0.8.3

### PR DESCRIPTION
**Pull request by wenmin92**
[Gridea](https://gridea.dev/en/): A static blog writing client.

License: MIT

### CODE REVIEW

File: bucket/gridea.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://gridea.dev/",
```
Comment (wenmin92) [inline bucket/gridea.json]:
```suggestion
    "homepage": "https://gridea.dev",
```

File: bucket/gridea.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://gridea.dev",
```
Comment (niheaven) [inline bucket/gridea.json]:
```suggestion
    "homepage": "https://gridea.dev/en/",
```

File: bucket/gridea.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        [
+            "Gridea.exe",
+            "Gridea"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/getgridea/gridea"
+    },
+    "autoupdate": {
+        "url": "https://github.com/getgridea/gridea/releases/download/v$version/Gridea.Setup.$version.exe#/dl.7z"
```
Comment (niheaven) [inline bucket/gridea.json]:
```suggestion
        "url": "https://github.com/getgridea/gridea/releases/download/v$version/Gridea.Setup.$version.exe#/dl.7z",
        "hash": {
            "url": "$baseurl/latest.yml",
            "regex": "sha512: $base64"
        }
```

File: bucket/gridea.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "version": "0.8.3",
+    "license": "MIT",
+    "url": "https://github.com/getgridea/gridea/releases/download/v0.8.3/Gridea.Setup.0.8.3.exe#/dl.7z",
+    "hash": "b499b43a14c6b32a447e40527415558806dc16b3bd241f266c10e7f60aacca25",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+        ]
+    },
```
Comment (niheaven) [inline bucket/gridea.json]: Move to `archi.64bit`.

**Pull request merged**